readthedocs/templates/projects/notifications/deprecated_config_file_used_email.html

```diff
@@ -13,7 +13,8 @@
     <li><strong>Monday, September 25, 2023</strong>: Fully remove support for building documentation without configuration file v2.</li>
 </ul>
 
-We have identified that the following projects which you maintain are impacted by this deprecation:
+We have identified that the following projects which you maintain, are impacted by this deprecation.
+Note that we changed our policy for sending this deprecation email to only projects with a successful build in the last year.
```

Comment:
```suggestion
We have identified that the following projects which you maintain, and were built in the last year, are impacted by this deprecation.
```

Perhaps this to be explicit?

Reply:
This was my first approach, but then I thought it was better if we explicitly communicate the change due to https://github.com/readthedocs/readthedocs.org/pull/10439#discussion_r1230782195